src/Common/Commands.Common.Authentication/AzureSessionInitializer.cs

```diff
@@ -183,15 +183,15 @@ public AdalSession()
             {
             }
 
-            public override TraceLevel AuthenticationLegacyTraceLevel
+            public override System.Diagnostics.TraceLevel AuthenticationLegacyTraceLevel
```

Comment:
Is this necessary?  Is this class in a different namespace in NetCore or something?

Reply:
Yes, Newtonsoft.

```
'TraceLevel' is an ambiguous reference between 'System.Diagnostics.TraceLevel' and 'Newtonsoft.Json.TraceLevel'
```

Reply:
Won't fix.